pkgs/development/python-modules/flask-webtest/default.nix

```diff
@@ -0,0 +1,58 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  flask,
+  webtest,
+  blinker,
+  flake8,
+  twine,
+  flask-sqlalchemy,
+  greenlet,
+}:
+
+buildPythonPackage rec {
+  pname = "flask-webtest";
+  version = "0.1.4";
+  pyproject = true;
+
+  # Pypi tarball doesn't include version.py
+  src = fetchFromGitHub {
+    owner = "level12";
+    repo = "flask-webtest";
+    rev = version;
+    hash = "sha256-4USNT6HYh49v+euCePYkL1gR6Ul8C0+/xanuYGxKpfM=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    flask
+    webtest
+    blinker
+  ];
+
+  nativeCheckInputs = [
+    flake8
+    twine
+    flask-sqlalchemy
+    greenlet
+  ];
+
+  checkPhase = ''
+    python3 setup.py sdist
+    twine check dist/*
```

Comment:
That's pretty useless: https://twine.readthedocs.io/en/stable/index.html#twine-check
```suggestion
```

Reply:
Okay, so `sdist` doesn't check anything useful either? I basically copied stuff from their `tox.ini` there.